### ISSUE TITLE
Composer installation

### ISSUE THREAD

**Issue by kpitn** [webburza/sylius-wishlist-bundle#7]
Hi, just try your last version (v0.4.0)

When using composer require i got this error : 

```
Runtime Notice: Webburza\Sylius\WishlistBundle\Model\Wishlist and Gedmo\Tim  
  estampable\Traits\Timestampable define the same property ($createdAt) in th  
  e composition of Webburza\Sylius\WishlistBundle\Model\Wishlist. This might   
  be incompatible, to improve maintainability consider using accessor methods  
   in traits instead. Class was composed   
```

Do you need to remove : createdAt and updatedAt ?

I don't know if it's already ready

**Comment by ychanan** [comment]
Resolved in v0.4.1, please try it again. Tnx.

**Comment by kpitn** [comment]
Installation is ok.

I will test the bundle and make you feedback if I find error.
